sdk/batch/Microsoft.Azure.Batch/tests/IntegrationTestCommon/IntegrationTestCommon.cs

```diff
@@ -100,8 +99,11 @@ private static CertificateCredentials GetBatchTestTenantCloudCredentials()
                     store.Close();
                 }
             }
-
+#if FullNetFx
             return new CertificateCredentials(certificate2);
```

Comment:
CertificateCredentials does not work with .NetCore2.1 can you suggest how to get around this?

Reply:
Do you know what the alternative is for CertificateCredentials in netcore? CertificateCredentials AFAIK comes from the Azure SDK so presumably there's some replacement for it?

Reply:
I haven't found an alternative foe that.